From bc17ee1a8d7ea87e7e3bb137c86519209eb24c00 Mon Sep 17 00:00:00 2001
From: Michael Tremer <michael.tremer@ipfire.org>
Date: Sat, 30 Jun 2012 12:18:52 +0200
Subject: [PATCH 13/16] Fix linking to libsolvext.

This is pulled in by libsolv itself. Although this is not good
style, it works more reliable without patching libsolvext which
is missing symbols from libexpat, libz and more.
---
 python/src/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/src/Makefile b/python/src/Makefile
index 583cc16..2741fa2 100644
--- a/python/src/Makefile
+++ b/python/src/Makefile
@@ -7,7 +7,7 @@ MODULE_LZMA    = _lzma.so
 MODULES        = $(MODULE_PAKFIRE) $(MODULE_LZMA)
 
 # Libs that are to be linked into the module.
-MODULE_PAKFIRE_LIBS = -lcap -lpython$(PYTHON_VERSION) -lsolv -lsolvext
+MODULE_PAKFIRE_LIBS = -lcap -lpython$(PYTHON_VERSION) -lsolv
 MODULE_LZMA_LIBS    = -llzma
 
 SOURCES_LZMA     = _lzmamodule.c
-- 
1.7.10.4

